YSE_App/yse_utils/get_YSE_PAs.py

```diff
@@ -358,7 +359,7 @@ def get_requested_fields(self):
                     field_msbs = np.append(field_msbs,s['field_id'].split('.')[0])
                     ras = np.append(ras,s['ra_cen'])
                     decs = np.append(decs,s['dec_cen'])
-
+                    priorities = np.append(priorities,d['priority'])
```

Comment:
Hey Dave -- can you please wrap the key dereference in a try-catch in case the 'priority' field isn't in the data?